app/src/main/res/layout/activity_main.xml

```diff
@@ -39,7 +39,7 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:gravity="center"
-                android:text="Sveiki atvykę"
+                android:text="Welcome"
```

Comment:
I see you have moved strings that are below to `strings.xml`, try that all string values would be in `strings.xml`